modules/architecture-rhcos-updating-bootloader.adoc

```diff
@@ -77,15 +78,15 @@ Updated: grub2-efi-x64-1:2.04-31.fc33.x86_64,shim-x64-15-8.x86_64
 ----
 
 .Machine config method
-Another way to enable `bootupd` is by providing a machine config.
+Another way to automatically update the boot loader during the boot process using `bootupd` is by providing a machine config.
 
-* Provide a machine config file with the enabled `systemd` unit, as shown in the following example:
+* Provide a machine config file with the `systemd` unit, as shown in the following example:
 +
 .Example output
-[source,yaml]
+[source,yaml,subs="attributes+"]
 ----
   variant: rhcos
-  version: 1.1.0
+  version: v{MaistraVersion}
```

Comment:
This has been replaced by `v2.4` and should be `v4.15.0` instead so this is probably not the right attribute.

Reply:
if this is the RHCOS version then `{product-version}` ?

Reply:
It's the Butane spec version. I'll have to dig into other Butane example to see what we use.

Reply:
Hum, my bad, it's not a Butane config but directly a MachineConfig one. Will have to look again.